packages/state-transition/src/block/processRandao.ts

```diff
@@ -1,11 +1,19 @@
-import xor from "buffer-xor";
 import {digest} from "@chainsafe/as-sha256";
 import {allForks} from "@lodestar/types";
 import {EPOCHS_PER_HISTORICAL_VECTOR} from "@lodestar/params";
 import {getRandaoMix} from "../util/index.js";
 import {verifyRandaoSignature} from "../signatureSets/index.js";
 import {CachedBeaconStateAllForks} from "../types.js";
 
+function xor(array1: Uint8Array, array2: Uint8Array): Uint8Array {
```

Comment:
calling those arrays is bit strange to me, maybe call them `b1` and `b2` (for bytes) or for simplicity just `a` and `b`